src/test/java/com/microsoft/sqlserver/jdbc/TestResource.java

```diff
@@ -0,0 +1,172 @@
+            {"R_parrentLoggerNameWrong", "Parent Logger name is wrong"},
+            {"R_unexpectedWrongDB", "Unexpected: ClientConnectionId is not in exception message due to wrong DB"},
+            {"R_unexpectedWrongHost", "Unexpected: ClientConnectionId is in exception message due to wrong host"},
+            {"R_cannotOpenDatabase", "Cannot open database"},
+            {"R_shouldNotConnect", "Should not have connected"},
+            {"R_loginFailed", "Login failed"},
+            {"R_exitedMoreSeconds", "Exited in more than {0} seconds."},
+            {"R_exitedLessSeconds", "Exited in less than {0} seconds."},
+            {"R_invalidArgumentExecutor", "The argument executor is not valid"},
+            {"R_threadInterruptNotSet", "Thread's interrupt status is not set."},
+            {"R_connectMirrored", "Connecting to a mirrored"},
+            {"R_trustStorePasswordNotSet", "The DataSource trustStore password needs to be set."},
+            {"R_invalidObjectName", "Invalid object name"},
+            {"R_tempTAbleNotRemoved", "Temporary table is not removed."},
+            {"R_firstConnectionNotClosed", "First connection is not closed"},
+            {"R_connectionNotClosedWithPoolClose", "Connection is not closed with pool close"},
+            {"R_connectionNotClosedWithPoolClose", "Unexpected: ClientConnectionId is null from Pool"},
+            {"R_idFromPoolNotSame", "ClientConnection Ids from pool are not the same."},
+            {"R_noProtocolVersion", "protocol version is not enabled or not supported by the client."},
+            {"R_protocolException", "Any protocol other than TLSv1, TLSv1.1, and TLSv1.2 should throw Exception"},
+            {"R_invalidProtocolLabel", "SSL Protocol {0} label is not valid. Only TLS, TLSv1, TLSv1.1, and TLSv1.2 are supported."},
+            {"R_SQLServerResourceMessage", "Message should be from SQL Server resources: "},
+            {"R_shouldThrowException", "Exception should have been thrown"},
+            {"R_tcpipConnectionToHost", "The TCP/IP connection to the host"},
+            {"R_queryTimedOut", "The query has timed out."},
+            {"R_readTimedOut", "Read timed out"},
+            {"R_unexpectedErrorMessage", "Unexpected error message occured!"},
+            {"R_warningsNotFound", "Warnings not found!"},
+            {"R_warningsFound", "Warnings found!"},
+            {"R_causeShouldNotBeNull", "Cause should not be null."},
+            {"R_causeShouldBeInstance", "Cause should be instance of {0}."},
+            {"R_connShouldNotBeClosed", "Connection should not be closed"},
+            {"R_connShouldNotBeOpen", "Connection should not be open"},
+            {"R_incorrectDriverNameFormat", "Driver name is not a correct format! "},
+            {"R_incorrectDriverVersionFormat", "Driver version number should be four parts! "},
+            {"R_previousShouldThrow", "Previous should have thrown an exception"},
```

Comment:
Most of these messages are very specific and I don't see them being used in any other test.